fv3config/config/rundir.py

```diff
@@ -16,6 +17,8 @@ def write_run_directory(config, target_directory):
         target_directory (str): target directory, will be created if it does not exist
     """
     logger.debug(f"Writing run directory to {target_directory}")
+    if "gfs_analysis_data" in config:
```

Comment:
This seems fine. But would it be better to switch this on based on the value of the `fv_core_nml.nudge` namelist option? Then, the base_yaml could have `gfs_analysis_data` default options without triggering a download for non-nudging runs.

Reply:
Went back and for on this, yeah I think you're right (switching on `fv_core_nml.nudge` is how the CLI behaves). The `write_run_directory` will fail if `fv_core_nml.nudge` is True and the user does not provide a `gfs_analysis_data` section, which is good.